Where_Is_My_Wife/Assets/_Main/Scripts/Player/Properties/PlayerProperties.cs

```diff
@@ -1,4 +1,5 @@
 using UnityEngine;
+using UnityEngine.Serialization;
```

Comment:
Erase this.

Reply:
5f0fa22d3ac664225c58d59713d18a187a8ef0c3